awx/main/management/commands/list_custom_venvs.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 Ansible, Inc.
+# All Rights Reserved
+import sys
+
+from awx.main.utils.common import get_custom_venv_choices
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+
+class Command(BaseCommand):
+    """Returns a list of custom venv paths from the path passed in the argument"""
+
+    def handle(self, *args, **options):
+        super(Command, self).__init__()
+        venvs = get_custom_venv_choices()
+        if venvs:
+            print('# {}'.format("Discovered virtual environments:"))
+            for venv in venvs:
+                print(venv)
+
+            msg = [
+                '',
+                'To export the contents of a (deprecated) virtual environment, ' 'run the following command while supplying the path as an argument:',
+                'awx-manage export_custom_venv /path/to/venv',
```

Comment:
Also mention the command to get the uses of that venv, and give the syntax that will work for that.